src/Integration.Vsix.UnitTests/Helpers/StatusBarNotifierTests.cs

```diff
@@ -19,32 +19,38 @@
  */
```

Comment:
This test class I rewrote mostly as the TestInitializer wasnt needed anymore and this way it was easier for me to add more tests.